File: areTheNumbersInOrder.cs
using System;

public class Kata
{
  public static bool IsAscOrder(int[] arr)
  {
      for (var i = 0; i < arr.Length - 1; i++)
      {
        if (arr[i] > arr[i + 1])
          return false;
      }
      return true;
  }
}

File: Java/areTheNumbersInOrder.java
public class Solution {

  public static boolean isAscOrder(int[] arr) {
    if (arr.length == 1)
      return true;
      
    for (int i = 0; i < (arr.length - 1); i++) {
      if (arr[i] > arr[i + 1]) {
        return false;
      }
    } 
      return true;
  }
}